lib/tbd/ua.rb

```diff
@@ -106,10 +106,11 @@ def uo(model = nil, lc = nil, id = "", hloss = 0.0, film = 0.0, ut = 0.0)
         loss = (new_u - k / d) * area
       end
 
-      ok = m.setThickness(d)
-      return invalid("Can't uprate #{id}: > 3m", mth, 0, ERR, res) unless ok
-
-      m.setThermalConductivity(k) if ok
+      if m.setThickness(d)
+        m.setThermalConductivity(k)
+      else
+        return invalid("Can't uprate #{id}: #{d} > 3m", mth, 0, ERR, res)
```

Comment:
MAX 3m limitation for material thickness [deprecated](https://github.com/NREL/OpenStudio/pull/4622) as of v3.5.X.